roles/sushy_emulator/tasks/create_container.yml

```diff
@@ -42,14 +42,8 @@
       - "{{ cifmw_sushy_emulator_sshkey_path }}:/root/.ssh/id_rsa:ro,Z"
       - "{{ cifmw_sushy_emulator_sshkey_path }}.pub:/root/.ssh/id_rsa.pub:ro,Z"
 
-# TODO(hjensas): We should probably change to use a DNS name once #1777 has landed.
 - name: Set Sushy Emulator URL for Podman installation
   vars:
-    _ip_address: >-
-      {%- if cifmw_sushy_emulator_listen_ip in ['::', '0.0.0.0'] -%}
-      {{ hostvars['controller-0'].host_ip | default(hostvars['controller-0']['ansible_host']) }}
-      {%- else -%}
-      {{ cifmw_sushy_emulator_listen_ip }}
-      {%- endif -%}
+    _listen: "{{ cifmw_sushy_emulator_listen_ip | ansible.utils.ipwrap }}"
   ansible.builtin.set_fact:
-    _sushy_url: "http://{{ _ip_address | ansible.utils.ipwrap }}:8000"
+    _sushy_url: "http://{{ _listen }}:8000"
```

Comment:
I would love to see a CNAME for this, `sushy-emaulator.utility` - then we can drop setting this fact.
... but that should be follow up. :smile:

Reply:
Yes that would be a great improvment on this already great improvment. Nice and clean!

Reply:
hmm like a `cifmw_sushy_emulator_address` ? To keep it working outside of the reproducer role. I can check that, once the utility zone is accepted (see other comment).

Reply:
I introduced a couple of parameters there to make things "nice" Lemme know your thoughts.